Report for GSOC 2019.md

```diff
@@ -4,7 +4,28 @@ This page is the location for the reporting related to the GSoC 2019 project tit
 
 Student: [Harman Deep Singh](https://github.com/hdsingh) Mentors: [Rich Signell](https://github.com/rsignell-usgs), [Martin Durant](https://github.com/martindurant) ESIP POC: [Annie Bryant Burgess](https://github.com/abburgess)
 
-## Community Bonding Period (May 6th - May 26th):
+Project Repository: [intake/xrviz](https://github.com/intake/xrviz)
+
+Project Documentation: [xrviz.readthedocs.io](https://xrviz.readthedocs.io)
+
+## Index
```

Comment:
Maybe a good idea to include a short summary at the top: GSOC project with.., done in the period..., with initial proposal at...,